### ISSUE TITLE
Customnametag and Multiple Lines Not Showing

### ISSUE THREAD

**Issue by Chociily** [NEZNAMY/TAB#839]
### Server version

Purpur 1.19.3

### TAB version

3.3.0

### Describe the bug

It should show my nickname with how I have it set up. I tried customtabname and it shows my essentials nickname, but not with customtagname. Also, when I go afk, it doesn't show on a line above my name. However, if I put %afk% on the same line as the rest of my nametag, it shows fine.
![image](https://user-images.githubusercontent.com/54920057/222133752-9da83b5b-9105-4e01-94fc-fa881406ce1d.png)

I can also set it to random gibberish and it still shows my normal name.
![image](https://user-images.githubusercontent.com/54920057/222134719-336cf2cf-bafe-40b4-a239-f2828ba5fdf8.png)
![image](https://user-images.githubusercontent.com/54920057/222134701-36b27e34-181e-416e-9819-46b06d16bcb6.png)

And yes, I have dynamic nametags enabled.
![image](https://user-images.githubusercontent.com/54920057/222134257-126fa840-4ea7-4385-b64f-0628737e699e.png)

### Expected behavior

Anything I put as my customnametag should show correctly, but instead, it stays default. It should also be able to show different lines of text rather than just one.

### Steps to reproduce

Simply put anything as your customnametag or try putting some text on another line.

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [ ] I ticked all of the boxes without actually reading them
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by NEZNAMY** [comment]
Show output of `/tab debug <player>`. Also, are you using F5 by accident?

**Comment by Chociily** [comment]
I'm using the freecam mod to show my nametag.
![image](https://user-images.githubusercontent.com/54920057/222135471-6fae67aa-3315-4eab-99a1-1eec3c1d3641.png)

**Comment by NEZNAMY** [comment]
https://github.com/NEZNAMY/TAB/wiki/Feature-guide:-Unlimited-nametag-mode#additional-note-2---f5-view same thing.
